airflow/post_install.sh

```diff
@@ -31,6 +31,8 @@ python $AIRFLOW_HOME/utils/set_default_variables.py \
     vessel_info_table="vessel_info" \
     temp_bucket="{{ var.value.TEMP_BUCKET }}" \
     window_days="30" \
+    max_gap_size_value="24" \
```

Comment:
@andres-arana you have added this variable but you are not using it anywhere in the code. Should it be deleted or you forgot to add the code that uses it?

Reply:
Just to inform @enriquetuya that the variable is used in https://github.com/GlobalFishingWatch/pipe-segment/blob/master/airflow/pipe_segment_dag.py#L38 
I don't see that is necessary to add it in the `post_install` script.
It was mentioned once that it's better that the dags break in case the variables is not defined. That is why it was added directly to the DAG without creating the Airflow Variable when the DAG is installed.

Reply:
That's true, I'll remove it.